site/content/docs/api-checks/timeouts-timing.md

```diff
@@ -17,3 +17,12 @@ The timing phases correspond to the Node.js request library timing phases:
 - `download`: Duration of HTTP download
 
 ![api monitoring timing phases](/docs/images/api-checks/timing-phases.png)
+
+## ESOCKETTIMEDOUT
```

Comment:
@josep-fl I see this is not wrapped in the callout. Is this on purpose?

Reply:
I thought being longer could fit in a category on its own. But a callout is actually better. Could also do a quote? 
![image](https://user-images.githubusercontent.com/26063772/145893078-3c66419b-8d57-4072-af0c-a02ee0b7fbf0.png)